docs/Developing.md

```diff
@@ -132,14 +132,81 @@ or visit `http://localhost:3000/ui/` direct.
 
 xpcshell and mochitest
 ----------------------
-
 These are tests for the [chrome](https://developer.mozilla.org/docs/Glossary/Chrome)
 modules and files.
 
-Currently these are not runnable in the github repository, and an export to
-mozilla-central is required. We are
-[working on improving](https://bugzilla.mozilla.org/show_bug.cgi?id=1238570) this
-situation.
+**Test Export to Mozilla Central(gecko-dev)**
+
+The gecko-dev repo is cloned locally and is used as the xpcshell and 
+mochitest test build.
+
+**Resetting gecko-dev repo for pull and export**
```

Comment:
I would just refer out to the appropriate instructions for updating the repository. Maybe just include instructions for git & mercurial only for resetting the repo.